eth/protocols/eth/protocol.go

```diff
@@ -128,8 +128,8 @@ type GetBlockHeadersPacket struct {
 
 // GetBlockHeadersPacket66 represents a block header query over eth/66
 type GetBlockHeadersPacket66 struct {
-	RequestId uint64
-	*GetBlockHeadersPacket
+	RequestId              uint64
+	*GetBlockHeadersPacket `rlp:"nil"`
```

Comment:
This is incorrect, the contained message should never be nil during decoding. There is a use of this type in tests where this field is nil for encoding, but it's OK and the test works as-is.